### DISCUSSION TITLE
whitelisting url

### DISCUSSION THREAD

**Discussion by KlentyAswinjoseph** [deepgram/community#680] [General help] [answered]
Is it feasible to add a URL to a whitelist for live transcription? With deepgram react sdk, I'm utilising

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by KlentyAswinjoseph** [reply]
All I'm requesting is the possibility.

**Comment by jpvajda** [comment, marked as answer]
@KlentyAswinjoseph just to confirm, are you asking about our Javascript SDK? We currently don't have an official React SDK.
